Christopher-christpheranderson/homework-1/bottles of beer assignment/js/main.js

```diff
@@ -0,0 +1,26 @@
+for (i = 99; i >= 1; i = i - 1){
+if (i <= 99){
+  document.write("")
```

Comment:
Nice job getting this done by any means necessary! And apologies for the directions being unclear. I was thinking `console.log` when I talked about the output being visible in the terminal.

Reply:
Glad to see the indents of all the statements in the `if` clause. In general, you want to indent each time you start a new nested section -- so in this case, you'd want to indent both `if` clauses within the `for` statement as well.